lib/hammer_cli_foreman/table_preferences.rb

```diff
@@ -0,0 +1,71 @@
+module HammerCLIForeman
+  class TablePreferences < HammerCLIForeman::Command
+    resource :table_preferences, :users
+    command_name "table-preferences"
+
+    class ListCommand < HammerCLIForeman::ListCommand
+      output do
+        field :id, _("Id")
+        field :name, _("Name")
+        field :columns, _("Columns")
+        field :created_at, _("Created at"), Fields::Date
+      end
+
+      build_options
+    end
+
+    class CreateTablePreferences < HammerCLIForeman::CreateCommand
```

Comment:
Let's stick with our conventions:
```suggestion
    class CreateCommand < HammerCLIForeman::CreateCommand
```